sideboard/websockets.py

```diff
@@ -586,8 +636,10 @@ def unsubscribe(self, clients):
     def unsubscribe_all(self):
         """Called on close to tear down all of this websocket's subscriptions."""
         for clients in self.subscriptions.values():
-            for client in clients.pop(self, {}):
-                self.teardown_passthru(client)
+            clients.pop(self, {})
+
+        for passthru_client in list(self.passthru_subscriptions.keys()):
+            self.teardown_passthru(passthru_client)
```

Comment:
This fixes a bug in our "passthru subscriptions" functionality; previously we would only clean up passthru subscriptions which were present in ``self.subscriptions`` instead of cleaning up all passthru subscriptions regardless.